src/main/java/Duke.java

```diff
@@ -1,10 +1,165 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    //CONSTANTS
+    public static final String SEPARATOR = "____________________________________________________________\n";
+
+    //METHODS
+    public static void printGreeting() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n"
+                + logo
+                + SEPARATOR
+                + " Hello! I'm Duke\n"
+                + " What can I do for you?\n"
+                + SEPARATOR);
+    }
+
+    public static void printError() {
+        System.out.println(SEPARATOR
+                + " Oops, something went wrong!\n"
+                + SEPARATOR);
+    }
+
+    public static String[] organize(String userInput) {
+        String[] organizedUserInput = new String[3];
+        if (!userInput.contains(" ")) {
+            //command is list
+            organizedUserInput[0] = userInput;
+        } else {
+            int spaceIndex = userInput.indexOf(" ");
+            String command = userInput.substring(0, spaceIndex);
+            organizedUserInput[0] = command;
+            if (!userInput.contains("/")) {
+                //command is either done or todo
+                String detail = userInput.substring(spaceIndex + 1);
+                organizedUserInput[1] = detail;
+            } else {
+                //command is either deadline or event
+                int slashIndex = userInput.indexOf("/");
+                String detail = userInput.substring(spaceIndex + 1, slashIndex - 1);
+                String time = userInput.substring(slashIndex + 1);
+                organizedUserInput[1] = detail;
+                organizedUserInput[2] = time;
+            }
+        }
+        return organizedUserInput;
+    }
+
+    public static void printTaskList(Task[] taskList, int taskListSize) {
+        System.out.println(SEPARATOR
+                + " Here are the tasks in your list:");
```

Comment:
I feel there is no need to wrap this sout statement into 2 lines, as it is a short statement. This affects readability